openday_scavenger/views/game/static/index.html

```diff
@@ -103,6 +103,11 @@ <h1 class="type-h1 mb-3">🗝 Unlock the Secrets of the 🗝<br />🕵️‍♂
         // for example:
     }
     document.addEventListener('DOMContentLoaded', function () {
+        if (!document.getElementById('#reader')) {
+            // When there is no session, the reader element is not rendered
+            return;
+        }
+
```

Comment:
This was causing a crash that didn't really do anything in the browser but it crashed the e2e tests